PaolaSantolloV/react-apprenticeship-capstone4
# Feat/final deliverable

## Description
This PR contain the final deliverable.

- Add some test to the components
- Fix the issue in the Cart

## Type of Change


## How should this be tested?
In your terminal run `npm install` after that `npm run start`, now you can see the page on[ localhost:3000](https://localhost:3000/)
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## What is the deliverable?
Final deliverable

##Screenshots and links
netlify link : [https://nostalgic-archimedes-7c817a.netlify.app/](https://nostalgic-archimedes-7c817a.netlify.app/)

## Checklist
- [x ] I added tests to prove that my fix is effective or my deliverable works.
